trulens_eval/trulens_eval/feedback/groundtruth.py

```diff
@@ -92,3 +116,89 @@ def agreement_measure(
             ret = np.nan
 
         return ret
+    
+    def bert_score(
+        self, prompt: str, response: str
+    ) -> Union[float, Tuple[float, Dict[str, str]]]:
+        """
+        Uses BERT Score. A function that that measures
+        similarity to ground truth using bert embeddings. 
+
+        Parameters:
+            prompt (str): A text prompt to an agent. response (str): The
+            agent's response to the prompt.
+
+        Returns:
+            - float: A value between 0 and 1. 0 being "not in agreement" and 1
+                being "in agreement".
+            - dict: with key 'ground_truth_response'
+        """
+        if self.bert_scorer is None:
+            self.bert_scorer = BERTScorer(lang="en", rescale_with_baseline=True)
```

Comment:
why does bert_scorer need to be a self property?

Reply:
Because each time we initialize a BERTScorer, this loads a large-ish BERT model from disk. This makes it quite expensive to do on each  feedback function call.

Reply:
got it. Can we put that in a comment in the code so its documented? This PR should be fine; as its loaded in memory, but it could pop up in the future as any property on feedback functions get serialized to DB (of which this is probably not serializable). Just want to make sure any dev has context and doesn't remove it if something does pop up

Reply:
I can do that. Also open to ideas on how to make this serializable.